packages/components/src/components/hds/code-block/index.ts

```diff
@@ -50,7 +50,7 @@ export interface HdsCodeBlockSignature {
     isStandalone?: boolean;
     language?: HdsCodeBlockLanguages;
     maxHeight?: string;
-    value: string;
+    value?: string;
```

Comment:
Issue: Another arg I think should be required.. in the code getter we check that value is not undefined.

Reply:
cool, I will update the docs and mark it as required then

Reply:
updated